katello-installer/katello-installer-base.spec

```diff
@@ -99,6 +105,9 @@ sed -ri 'sX\:installer_dir.*$X:installer_dir: %{_datadir}/katello-installer-base
 sed -ri 'sX\:module_dirs.*$X:module_dirs: \["%{_datadir}/katello-installer-base/modules", "%{_datadir}/foreman-installer/modules"\]Xg' config/*.yaml
 sed -ri 'sX\:hook_dirs.*$X:hook_dirs: \["%{_datadir}/katello-installer-base/hooks"\]Xg' config/*.yaml
 
+# generate parser-cache
+kafo-export-params -c config/katello.yaml -f parsercache --no-parser-cache -o %{_datadir}/foreman-installer/katello.yaml
```

Comment:
We have a similar rake task in our katello-installer so I would think it should be added there, the only difference is they run the rake build in the RPM spec, and we do it out of band somewhere else (jenkins?)